src/Air.zig

```diff
@@ -356,9 +356,9 @@ pub const Inst = struct {
         /// Base 10 logarithm of a floating point number.
         /// Uses the `un_op` field.
         log10,
-        /// Aboslute value of a floating point number.
+        /// Aboslute value of an integer, floating point number or vector.
         /// Uses the `un_op` field.
```

Comment:
Please document that this now uses `ty_op`, and that the result type may be different from the operand type for integers, which should be treated as an unsafe intcast. I know the documentation of AIR instructions is a bit poor in places, but it's good to at least make it precise when you change stuff!